lambdalayer/README.md

```diff
@@ -4,13 +4,15 @@
 
 ### Prerequisites
 
-Ensure you run the latest VPC stack and SAM Deployment Pipeline stack versions. At time of writing, v1.14.3 and v2.36.0, respectively. You can check the version of stacks in the [CloudFormation Console](https://eu-west-2.console.aws.amazon.com/cloudformation/home?region=eu-west-2#/stacks) in the description column.
+Ensure you run the latest VPC stack and SAM Deployment Pipeline stack versions. At time of writing, v2.0.2 and v2.36.0, respectively. You can check the version of stacks in the [CloudFormation Console](https://eu-west-2.console.aws.amazon.com/cloudformation/home?region=eu-west-2#/stacks) in the description column.
 
 If you are using Secure Pipeline to deploy your SAM application, you will need to make the following modifications:
 
 - `AdditionalCodeSigningVersionArns`: add `arn:aws:signer:eu-west-2:216552277552:/signing-profiles/DynatraceSigner/5uwzCCGTPq`
 - `CustomKmsKeyArns`: add `arn:aws:kms:eu-west-2:216552277552:key/4bc58ab5-c9bb-4702-a2c3-5d339604a8fe`
 
+Your lambda needs to have access to the internet, so that it can send it's metrics to Dynatrace.
```

Comment:
Just for consistency, can we say 'your function' rather than 'your lambda' (or 'your Lambda Function', I guess)

Reply:
sure